dotcom-rendering/src/components/CardPicture.tsx

```diff
@@ -139,8 +139,7 @@ export const CardPicture = ({
 				src={fallbackSource.lowResUrl}
 				css={block}
 				loading={loading}
-				// Chromatic inconsistenly loads the image for lazy-loaded images
-				data-chromatic={loading === 'lazy' ? 'ignore' : undefined}
+				data-chromatic="ignore"
```

Comment:
big & bold – love it!

Reply:
There'll be a separate PR for this change - I just wanted to see whether the Chromatic diffs would go away (they do)